ocis-pkg/sync/mutex_test.go

```diff
@@ -0,0 +1,33 @@
+package sync
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+)
+
+func HammerMutex(m *NRWMutex, loops int, cdone chan bool) {
```

Comment:
aaah, a man of culture! this is straight out of the stdlib! nice test :)

Reply:
exactly, it's borred 🤣